flyteidl/protos/flyteidl/admin/launch_plan.proto

```diff
@@ -144,6 +144,22 @@ message LaunchPlanSpec {
     // ClusterAssignment controls how to select an available cluster on which executions of this LaunchPlan should run.
     // This can be overwritten at execution creation level.
     ClusterAssignment cluster_assignment = 23;
+
+    // Concurrency setting for a given workflow to determine concurrency limits and behavior when the limit is breached
+    ConcurrencyPolicy concurrency_policy = 25;
```

Comment:
Should this be 24? I wonder if you have something already using 24 in your fork :P

Reply:
Ah good catch, changed to 24